bin/create

```diff
@@ -21,6 +21,8 @@
 var path = require('path');
 var create = require('./lib/create');
 var args  = require('./lib/simpleargs').getArgs(process.argv);
+var apk_url = args['--xwalk-apk-url'];
```

Comment:
It's probably safer to use `apk_url` and `shared_library` in the call to `createProjects` below. Even though it makes no difference now, it might confuse others in the future if we manipulate these variables and then pass the original values below.